### PR TITLE
Fix: shared postcard width

### PR DESCRIPTION
**Pull request by nensidosari**
## Changes

Potential solution for https://dailydotdev.slack.com/archives/C059GQ5RFP0/p1702912778957629

### CODE REVIEW

**Review by kopancek — APPROVED** [review]
LGTM :shipit:

File: packages/shared/src/components/cards/SharedPostCardFooter.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ export const SharedPostCardFooter = ({
   return (
     <div
       className={classNames(
-        'p-3 gap-3 mb-2 rounded-12 border border-theme-divider-tertiary flex w-fit flex-auto h-auto min-h-0',
+        'p-3 gap-3 mb-2 rounded-12 border border-theme-divider-tertiary flex w-fit flex-auto h-auto min-h-0 w-full',
```
Comment (rebelchris) [inline packages/shared/src/components/cards/SharedPostCardFooter.tsx:22]: There are contradicting widths now.
In our tailwind version only 1 will win and think it's currently based on alphabetical order, maybe double check which one get's rendered and delete the other?

Refering to `w-fit` and `w-full`

**Review by rebelchris — CHANGES_REQUESTED** [review]
One small request.

**Comment by omBratteng** [comment]
@capJavert you encountered this issue, could you please check if this fixes it?

File: packages/shared/src/components/cards/SharedPostCardFooter.tsx (hunk 1 above)
Comment (omBratteng) [reply packages/shared/src/components/cards/SharedPostCardFooter.tsx:22]: That's what's a bit annoying with using `classNames` function, the linter doesn't catch it 😬
That's why I (try to) always paste the classes in a regular `className` to get the linter to do its thing.

**Review by capJavert — APPROVED** [review]
I don't really see what the changes are because of rebase but when I throttle locally I can not see the jump anymore 👍

File: packages/shared/src/components/cards/SharedPostCardFooter.tsx (hunk 1 above)
Comment (nensidosari) [reply packages/shared/src/components/cards/SharedPostCardFooter.tsx]: Removed w-fit, that was my intention, just missed removing it 😓

**Review by omBratteng — APPROVED** [review]

**Review by rebelchris — APPROVED** [review]

**Pull request merged**